src/components/Assignment/index.ts

```diff
@@ -140,7 +139,6 @@ class Assignment extends BridgeBase {
                   .arMainButtons=${this.arMainButtons} .arSecondaryButtons=${this.arSecondaryButtons}
                   @AssignmentActionButtonClick="${this._onActionButtonClick}">
                 </assignment-card-component>
-                <lit-toast></lit-toast>
```

Comment:
Similarly, a comment here so we can remember everyplace where <lit-toast> used to be used.